### PR TITLE
Add hive metastore connectivity and permission check tool

### PR DESCRIPTION
**Pull request by LuQQiu**
Add a tool to check the connectivity and functionalities of an existing hive metastore.

### CODE REVIEW

**Comment by LuQQiu** [comment]
Jenkins, test this please

File: pom.xml [hunk 1]
```diff
@@ -133,6 +133,7 @@
     <netty.version>4.1.45.Final</netty.version>
     <rocksdb.version>5.15.10</rocksdb.version>
     <hadoop.version>2.7.3</hadoop.version>
+    <hive.version>2.3.6</hive.version>
```
Comment (madanadit) [inline pom.xml:136]: can we only support a specific hive client version? how did you choose 2.3.6, which apache hadoop or cdh versions is it compatible with

Comment (madanadit) [reply pom.xml:136]: @gpang can you please comment on the hive client requirementS

Comment (LuQQiu) [reply pom.xml:136]: based on @gpang  hive version capatibility testing, most operations are the same in 2.2.0 and 2.3.6, but 2.3.6 has a new get_table_request(which is not used by hms tool) which does not support older versions. Although this operation is not used by hms tool so far, to be safe, will change to 2.2.0 and tested it out.

**Comment by LuQQiu** [comment]
CLI tests failed consistently on cannot find test logs

concurrentListDirsNonPersistedDir(alluxio.client.fs.ConcurrentFileSystemMasterCreateTest)  Time elapsed: 1.764 sec  <<< ERROR!
java.lang.RuntimeException: java.io.FileNotFoundException: ./target/logs/tests-alluxio.client.fs.ConcurrentFileSystemMasterCreateTest-concurrentListDirsNonPersistedDir.log (No such file or directory)
Caused by: java.io.FileNotFoundException: ./target/logs/tests-alluxio.client.fs.ConcurrentFileSystemMasterCreateTest-concurrentListDirsNonPersistedDir.log (No such file or directory)

**Comment by LuQQiu** [comment]
jenkins, test this please

File: docs/en/operation/User-CLI.md [hunk 2]
```diff
@@ -246,6 +246,26 @@ $ ./bin/alluxio runTests
 The `runJournalCrashTest` simulates a failover to test recovery from the journal.
 Note that this command will stop any Alluxio services running on the machine.
 
+### runHmsTest
+
+The `runHmsTests` amis to validate the configuration, connectivity, and permissions of an existing hive metastore
```
Comment (jiacheliu3) [inline docs/en/operation/User-CLI.md:251]: `amis` -> `aims`?

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**